### PR TITLE
[PRM-628] Separated Call to Action Rows using Estimated/Due status

### PR DESCRIPTION
**Pull request by Chris-Williams-QA**
…T/Unit tests TBD

### CODE REVIEW

**Comment by Chris-Williams-QA** [comment]
Commit has an incorrect name, but regardless everything should be good to go.

File: it/controllers/IndexControllerISpec.scala [hunk 1]
```diff
@@ -555,22 +555,25 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
     }
 
     "return 200 (OK) and render the view when there are LPPs paid that are retrieved from the backend" in {
-      returnLSPDataStub(etmpPayloadWithLPP)
+      returnLSPDataStub(etmpPayloadWithPaidLPP)
       val request = await(buildClientForRequestToApp(uri = "/").get())
       request.status shouldBe Status.OK
       val parsedBody = Jsoup.parse(request.body)
+      println(parsedBody.select("#late-payment-penalties section").text)
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:562]: Could you remove this println?

File: it/controllers/IndexControllerISpec.scala [hunk 2]
```diff
@@ -555,22 +555,25 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
...
       parsedBody.select("#late-payment-penalties section header h3").text shouldBe "£400 penalty"
       parsedBody.select("#late-payment-penalties section header strong").text shouldBe "paid"
       val summaryCardBody = parsedBody.select(" #late-payment-penalties .app-summary-card__body")
+      println(summaryCardBody.text)
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:566]: Could you remove this println?

File: it/controllers/IndexControllerISpec.scala [hunk 3]
```diff
@@ -555,22 +555,25 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
...
       summaryCardBody.select("dt").get(1).text shouldBe "Penalty reason"
-      summaryCardBody.select("dd").get(1).text shouldBe "VAT not paid within 15 days"
+      summaryCardBody.select("dd").get(1).text shouldBe "VAT not paid within 30 days"
       parsedBody.select("#late-payment-penalties footer li").text() shouldBe "Appeal this penalty"
     }
-    
+
     "return 200 (OK) and render the view when there is outstanding payments" in {
-      returnLSPDataStub(etmpPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue.copy(latePaymentPenalties = latePaymentPenalty))
+      //returnLSPDataStub(etmpPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue.copy(latePaymentPenalties = paidLatePaymentPenalty))
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:575]: Could you remove this comment?

File: it/controllers/IndexControllerISpec.scala [hunk 4]
```diff
@@ -645,13 +648,14 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
       val request = await(buildClientForRequestToApp(uri = "/").get())
       request.status shouldBe Status.OK
       val parsedBody = Jsoup.parse(request.body)
+      println(parsedBody.select("#late-payment-penalties section").text)
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:651]: Could you remove this println?

File: it/controllers/IndexControllerISpec.scala [hunk 5]
```diff
@@ -719,18 +723,18 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
 
       "return 200 (OK) and render the view when there is outstanding payments for the client" in {
         AuthStub.agentAuthorised()
-        returnAgentLSPDataStub(etmpPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue.copy(latePaymentPenalties = latePaymentPenalty))
+        returnAgentLSPDataStub(etmpPayloadWithLPPVATUnpaidAndVATOverviewAndLSPsDue.copy(latePaymentPenalties = paidLatePaymentPenalty))
         val request = controller.onPageLoad()(fakeAgentRequest)
         await(request).header.status shouldBe Status.OK
         val parsedBody = Jsoup.parse(contentAsString(request))
+        println(parsedBody.select("#what-is-owed").text)
```
Comment (LewisMitchell) [inline it/controllers/IndexControllerISpec.scala:730]: Could you remove this println?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good, just a few bits to cleanup. Running AT's now

**Comment by LewisMitchell** [comment]
image

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**